### ISSUE TITLE
There is no way to delete a tile when the cardinality is "1"

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#2740]
When the cardinality of a card is "1", then there is no easy way delete the tile.
Users could try to simply clear the contents of each widget and then "save" the tile with the cleared contents but that doesn't actually delete the tile from the database, all that really does is save a tile with null values.

**Comment by chiatt** [comment]
In discussing this issue with @apeters we decided that any solution would not provide enough benefit to users to justify the effort.  However, if resolving this issue becomes more of a priority here are a few solutions we discussed:

1. Add a list to each datatype defining which values constitute a null value. Upon saving a tile, check if all nodes have null values and if so, delete it rather than save it.  Once that is done we could do one of the following:
    - A clear button could be added to each widget to clear the values of each widget before saving so that the tile is deleted on save.
    - Write a command that would search the database for empty tiles and delete them

2. Add a delete button to cards with a cardinality of 1 similar to cards with an 'n' cardinality. Clicking the button would simply delete the tile and reset the card.
